gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenViewModel.java

```diff
@@ -306,6 +306,7 @@ public void onFeatureClick(ImmutableList<MapFeature> mapFeatures) {
   }
 
   public ImmutableList<Layer> getModifiableLayers() {
+    Timber.v("getModifiableLayers for %s", getActiveProject());
```

Comment:
I don't see any methods that take `Project` directly. Are you referring to `addObservation()`, where we extract the project from the feature? Any idea how we might do this otherwise?

Reply:
That makes sense. There may not be a better way to do it. I'll think on it a little bit and send a PR if I come up with something (probably not though)